File: .old/v4/engine/ai/secretary.lua
local Secretary = setmetatable({}, ai.NPC)
Secretary.__index = Secretary

function Secretary:new(x, y)
   return ai.NPC.new(self, x, y)
end

-------------
-- PRIVATE --
-------------

function Secretary:__tostring()
   return '[Secretary]'
end

function Secretary:type() return 'Secretary' end

return Secretary

-- vim: ts=3:sw=3:sts=3:expandtab

File: engine/npc/secretary.cc
#include "engine/npc/secretary.h"

#include <memory>
using namespace std;

#include "./main.h"
#include "ui/ui.h"
#include "ui/dialogmanager.h"

Secretary::Secretary(Point init, class City& city)
    : Worker(init, city, 0)
{
}

void
Secretary::TalkToHero()
{
}


// vim: ts=4:sw=4:sts=4:expandtab
